openwisp_monitoring/device/static/monitoring/js/device-map.js

```diff
@@ -141,6 +142,17 @@ function onAjaxSuccess(data) {
                         '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
                 },
             },
+            {
+                label: 'Satellite',
+                urlTemplate:
+                    'https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}',
+                options: {
+                    minZoom: 3,
+                    maxZoom: 32,
+                    attribution:
+                        '&copy; Source: <a href="http://www.esri.com/">Esri</a> and the GIS User Community ',
+                },
+            },
```

Comment:
the tile configuration must be taken from the LEAFLET_CONFIG setting, we shall not duplicate this here.

Please see my previous review in which I provide a sample LEAFLET_CONFIG which I want to be added to the setting of the test project here.

https://github.com/openwisp/openwisp-monitoring/pull/433#pullrequestreview-1080746410